app/models/custom_notification.rb

```diff
@@ -0,0 +1,31 @@
+class CustomNotification < ApplicationRecord
+
+  belongs_to :organization
+
+  has_and_belongs_to_many :users
+  has_one :massive_job, as: :target
+  has_many :notifications, as: :target
```

Comment:
do we use these 2 other associations for anything?

Reply:
DONE!